app/ui/faviconProgressbar.js

```diff
@@ -0,0 +1,49 @@
+const { platform } = require('../utils');
+const assets = require('../../common/assets');
+
+module.exports.updateFavicon = function(percentageString) {
+  if (platform() === 'web') {
+    let progress = parseInt(percentageString.replace('%', ''));
+    if (progress === 0 || progress === 100) {
+      const link = document.querySelector("link[rel*='icon']");
+      link.type = 'image/png';
+      link.href = assets.get('favicon-32x32.png');
+      document.getElementsByTagName('head')[0].appendChild(link);
+      return;
+    }
+    progress = progress * 0.01;
+    const link = document.querySelector("link[rel*='icon']");
+    const canvas = document.createElement('canvas');
+    const size = 32;
+
+    const loaderWidth = 5;
+    const loaderColor = '#0090ed';
+
+    const span = document.createElement('span');
+    span.textContent = percentageString;
+    const context = canvas.getContext('2d');
+    canvas.width = canvas.height = size;
+
+    context.translate(size / 2, size / 2);
+
+    const radius = (size - loaderWidth) / 2;
```

Comment:
last thing from me...recommend you calculate radius inside your function since it's dependent on another property of the circle....this means you'll need to pass size into the function as well...
```
const drawCircle = function(color, lineWidth,OuterWidth,percent) {
const radius = (outerWidth - loaderWidth) / 2;
...
}

Reply:
yeah i messed that up...should be outerwidth - lineWidth